pkgs/tools/admin/lego/default.nix

```
@@ -2,18 +2,23 @@
 
 buildGoModule rec {
   pname = "lego";
-  version = "3.2.0";
+  version = "3.3.0";
 
   src = fetchFromGitHub {
     owner = "go-acme";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1djvwyjg30f9bj61pf3y2k2w055pj39v0sif4rjqg8cz0j382a2z";
+    sha256 = "135zz5gp5vqqwmz3701n5xfvz9yxzm4m53q3dbj9vfc8xkcxn44n";
   };
 
-  modSha256 = "0k3p11cji3p4nzr8aia8hp01wyx1qfx84259dwbfwg1b32ln8rkc";
+  modSha256 = "0jirpfd427317px0fd630bmi3li6zc5vihydwmwbj0qsfvhn4qm4";
   subPackages = [ "cmd/lego" ];
 
+  buildFlagsArray = ''
+    -ldflags=
+      -X main.version=${version}
+  '';
```

This is pretty confusing; how about something like:

```suggestion
  buildFlagsArray = [
    "-ldflags='-X main.version=${version}'"
  ];
```